USEPA/mywaterway
# Bugfix/601 increase modal max width

## Related Issues:
* [HMW-601](https://jira.epa.gov/browse/HMW-601)

## Main Changes:
* Increased the width of the characteristics modal by 1 unit: previously, when the scrollbar shown, the "View Water Monitoring Report" info box text wrapped so a single period was on the second line.
* Changed the modal close button to an icon, rather than an "x", due to spacing.

## Steps To Test:
1. View a characteristics modal for a location with enough characteristics to overflow (and a scrollbar to appear). Ensure the info box text does not wrap.
